src/components/ProductListing/ProductListing.jsx

```diff
@@ -0,0 +1,66 @@
+import { useProducts } from "../../context";
+
+const ProductListing = () => {
+  const {productsData} = useProducts();
+
+  return (
+    <article className="grid-col-70 ">
+      <div className="products-container flex-row align-center flex-gap-2 flex-wrap">
+        {productsData.length !== 0 ?  
+        productsData.map(({_id, name, brand, category, discount, discountPercent, imgURL, mrp, price, rating, reviews, type }) => {
```

Comment:
remove the `{ }` and `return`, Use the implicit return as you are return only the jsx